serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -203,6 +203,104 @@ public class MyInterfaceOrClass {
 Avoid using `java` functions to call the external services, instead, you can use the xref:service-orchestration/orchestration-of-openapi-based-services.adoc[services orchestration features].
 ====
 
+== Camel custom function
+
+Kogito supports the link:{camel_url}[Camel Routes] functions within an Apache Maven project, in which you define your workflow service.
+
+[TIP]
+====
+NOTE: This section briefly exemplifies how to define and use Camel Routes within your workflow application. For more information, see the xref:integrations/camel-routes-integration.adoc[Integrating with Camel Routes].
+====
+
+=== Function definition
+
+The following example shows the declaration of a Camel function:
+
+[source,json]
+----
+{
+  "functions": [
+    {
+      "name": "myCamelEndpoint", <1>
+      "type": "custom",  <2>
+      "operation": "camel:direct:myendpoint" <3>
+    }
+  ]
+}
+----
+
+<1> `myCamelEndpoint` is the function name
+<2> `custom` is the function type
+<3> `camel:direct:myendpoint` is the custom operation definition. In this definition, `camel` is the reserved keyword followed by the `direct` endpoint. link:{camel_extensions_url}/direct.html[Camel Direct] is the only supported consumer by {context}. Finally, `myendpoint` is the endpoint URI name found in the route within your project's context.
```

Comment:
same here, we should point to a camel supported components table IMO

Reply:
We don't need this table since this is the only component supported, and I don't see any other producer to be supported. If so, we can then create a table.

Reply:
ok